TremblingV5/DouTok
# enhance configurator

## Link to an issue



## What



enhance configurator

## How



Add a tag named `configPath` for each config struct. Which means where should we load configurations from a `yaml` file by using viper.

Clarify priorities of loading from env variables and loading from files. Files > Env.

## Screenshots



## How to test



1. Run related unit test
2. Run services which used this configurator and confirm we got configurations which we want.

## Checklist

- [ ] Link to an issue if it really related.
- [x] At least describe what this PR does.
- [x] Use `rebase` to confirm that current branch doesn't conflict with main branch.
- [x] Unit tests. At least do not reduce the unit test coverage.
- [ ] Checked and updated `guidelines.md` which used to describe how to build, deploy and use DouTok.
